test/AllTests.cpp

```diff
@@ -1,3 +1,4 @@
+#include <gmock/gmock.h>
```

Comment:
Why bringing in gmock everywhere? Seems not to be needed, right? 
I probably do not mind, but I am curious.